lib/inferno/apps/cli/main.rb

```diff
@@ -52,6 +53,11 @@ def suites
       subcommand 'suite', Suite
 
       register(New, 'new', 'new TEST_KIT_NAME', 'Run `inferno new --help` for full help')
+
+      desc 'version', "Output Inferno version (#{Inferno::VERSION})"
+      def version
+        puts Inferno::VERSION
```

Comment:
This seems like this should be a standalone PR that can get in real quick without any fuss, instead of mixing it in with this bigger PR which will likely require some conversation.  We already have a ticket for it too. See FI-2116

Reply:
Sounds good I'll pick up FI-2116 and put `inferno version` as a seperate PR first. It's essentially a prerequisite for this to work.